stingray/io.py

```diff
@@ -680,7 +687,7 @@ def _save_ascii_object(object, filename):
 def _retrieve_ascii_object(object, filename):
     pass
 
-def write(input, filename, format = 'pickle'):
+def write(input, filename, format = 'pickle', save_as_dict = True):
```

Comment:
general question: `read` and `write` are in the future going to have different keywords depending on which read/write method (pickle, ascii, fits ...) are used. Are we happy just piling them into `read` and `write` as keyword arguments or do we prefer going the `**kwargs` route, where `kwargs` would include the arguments for the various helper methods?